src/cellfinder_core/detect/filters/volume/structure_splitting.py

```diff
@@ -111,14 +111,14 @@ def iterative_ball_filter(
     return ns, centres
 
 
-def check_centre_in_cuboid(centre: Point, max_coords: np.ndarray) -> bool:
+def check_centre_in_cuboid(centre: np.ndarray, max_coords: np.ndarray) -> bool:
     """
     Checks whether a coordinate is in a cuboid
     :param centre: x,y,z coordinate
     :param max_coords: far corner of cuboid
     :return: True if within cuboid, otherwise False
     """
-    relative_coords = np.array([centre.x, centre.y, centre.z])
+    relative_coords = np.array([centre[0], centre[1], centre[2]])
```

Comment:
```suggestion
    relative_coords = centre
```
Casting to an array is now redundant as it's already an array